benchmarks/tests/splines.py

```diff
@@ -32,7 +33,7 @@ def _basis_funcs(self, x: 'float', span: 'int', values: 'float[:]'):
 
             values[j+1] = saved
 
-    def eval(self, x : 'const float[:]', y : 'float[:]'):
+    def eval(self, x : 'Final[float[:]]', y : 'float[:]'):
```

Comment:
I cannot remember, does `Final['float[:]']` work, too? I am asking because we import `Final`, but we do not really use it if we annotate with a string.